app-misc/freeplane/freeplane-1.6.15.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Use repoman to commit to fix this line, it's 'Authors' instead of 'Foundation' nowadays.